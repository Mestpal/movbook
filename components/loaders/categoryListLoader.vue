<template>
  <v-row>
    <cardLoad
      v-for="n in 20"
      :key="n"
    />
  </v-row>
</template>

<script>
import cardLoad from '@/components/atomics/loaders/cardLoad'

export default {
  components: {
    cardLoad
  }
}
</script>
