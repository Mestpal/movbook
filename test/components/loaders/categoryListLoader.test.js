
import Vue from 'vue'
import Vuetify from 'vuetify'
import Vuex from 'vuex'
import { shallowMount, createLocalVue } from '@vue/test-utils'

import categoryListLoader from '@/components/loaders/categoryListLoader'

Vue.use(Vuetify)
Vue.use(Vuex)
const localVue = createLocalVue()

describe('categoryListLoader.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(categoryListLoader, {
      localVue
    })
  })

  it('reders the categoryListLoader component', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
