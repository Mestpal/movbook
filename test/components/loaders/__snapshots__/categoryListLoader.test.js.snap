// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`categoryListLoader.vue reders the categoryListLoader component 1`] = `
"<v-row-stub tag=\\"div\\">
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
  <cardload-stub></cardload-stub>
</v-row-stub>"
`;
